res/rect.f.glsl

```diff
@@ -12,11 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 #version 330 core
-in vec4 color;
+
+uniform vec4 col;
```

Comment:
Super not-picky, but I'd prefer `color` unless it's like this for a reason I don't understand. Alacritty does a lot with columns (though to be fair, not in this context) and I generally like avoiding potential reading hazards.

Reply:
So do I.